testsuites/listener/shared/client_sg/listener_initial_replication.py

```diff
@@ -0,0 +1,93 @@
+from keywords.utils import log_info
+from keywords.utils import log_r
+from keywords.MobileRestClient import MobileRestClient
+from libraries.data import doc_generators
+
+
+def large_initial_pull_replication(ls_url, cluster_config, num_docs, continuous):
+
+    sg_db = "db"
+    ls_db = "ls_db"
+
+    sg_one_admin = cluster_config["sync_gateways"][0]["admin"]
+    sg_one_public = cluster_config["sync_gateways"][0]["public"]
+
+    log_info(ls_url)
+    log_info(sg_one_admin)
+    log_info(sg_one_public)
+
+    client = MobileRestClient()
+    client.create_user(sg_one_admin, sg_db, "seth", password="password", channels=["ABC", "NBC"])
+    session = client.create_session(sg_one_admin, sg_db, "seth")
+
+    # Create 10000 docs on sync_gateway
+    docs = client.add_docs(
+        url=sg_one_public,
+        db=sg_db,
+        number=num_docs,
+        id_prefix="seeded_doc",
+        generator="four_k",
```

Comment:
It will create 10000 docs with the same `_id` property, however, it is adding them to sync_gateway via `PUT /db/<doc_id>` where <doc_id> is `id_prefix + number`, so in this example the docs would be seeded_doc_0, seeded_doc_1 ... seeded_doc_9999